src/Console/ConsoleInput.php

```diff
@@ -55,20 +55,25 @@ public function __construct(string $handle = 'php://stdin')
     /**
      * Read a value from the stream
      *
-     * @return string The value of the stream
+     * @return string|null The value of the stream. Null on EOF.
      */
-    public function read(): string
+    public function read(): ?string
     {
         if ($this->_canReadline) {
             $line = readline('');
```

Comment:
PHP manual claims `readline()` always returns string https://www.php.net/manual/en/function.readline.php :angry:.

Reply:
That's right. The only reference I could find was this note: https://www.php.net/manual/en/function.readline.php#86719
Both readline and fgets claims to return only string.